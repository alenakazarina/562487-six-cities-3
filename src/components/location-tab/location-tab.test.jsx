import React from 'react';
import renderer from 'react-test-renderer';
import LocationTab from './location-tab';

const locations = [`Paris`, `Cologne`, `Brussels`, `Amsterdam`, `Hamburg`, `Dusseldorf`];

describe(`LocationTab`, () => {
  it(`should render LocationTab correctly`, () => {
    const tree = renderer.create(
        <LocationTab
          location={locations[1]}
          isActive={false}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`should render LocationTab with active class correctly`, () => {
    const tree = renderer.create(
        <LocationTab
          location={locations[0]}
          isActive={true}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
