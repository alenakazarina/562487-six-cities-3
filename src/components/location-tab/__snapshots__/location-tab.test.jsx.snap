// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LocationTab correctly 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item "
    href="#"
  >
    <span>
      Cologne
    </span>
  </a>
</li>
`;

exports[`should render LocationTab with active class correctly 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item tabs__item--active"
    href="#"
  >
    <span>
      Paris
    </span>
  </a>
</li>
`;
